ArmRobot/Assets/Scripts/Camera/RandomizerPositionCamera.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class RandomizerPositionCamera : MonoBehaviour
+{
+    public float changePositionCamera = 0.005f;
+    public float rotationCamera = 1.0f;
+
+    public Vector3 initialCameraPosition = new Vector3(0.0f, 1.8f, 0.0f);
+
+    public Vector3 initialCameraRotation = new Vector3(90.0f, 0.0f, 0.0f);
```

Comment:
Better! But wouldn't it be easier for the user to move the camera in the scene, and then have this set to that position automatically? So maybe leave these properties empty here (`public Vector3 initialCameraPosition;`) and then set them to current position in Start() (`initialCameraPosition = transform.position`). This way the user doesn't have to manually re-enter these numbers into this class every time they move the camera.